admin/src/api/post/content-types/post/lifecycles.js

```diff
@@ -68,11 +68,6 @@ async function modifyContentAndSetErrorMsg(event) {
     // validate input fields
     validateFields(result);
 
-    if (result.is_resource && result.title) {
```

Comment:
Will this automatically generate slug when not present?

Reply:
yes